spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) {User.create!(first_name:'s',last_name:'j', email: 'jay2@he.com',password:'foobar',password_confirmation:'foobar')}
+
+  it "expects post to be valid" do
```

Comment:
When we test for validity we don't create the object yet. After all, validations happen before the object is saved in the database. 

What we can do here is something like:

`post = build(:post)` using factories

Then if this factory is a valid post:
`expect(post).to be_valid`